examples/lavalink-basic-bot.rs

```diff
@@ -53,7 +53,7 @@ async fn main() -> anyhow::Result<()> {
         let lavalink = Lavalink::new(user_id, shard_count);
         lavalink.add(lavalink_host, lavalink_auth).await?;
 
-        let intents = Intents::GUILD_MESSAGES | Intents::GUILD_VOICE_STATES;
+        let intents = Intents::GUILD_MESSAGES | Intents::GUILD_VOICE_STATES | Intents::MESSAGE_CONTENT;
```

Comment:
No message content was in the message so the bot was broken when you sent a command. I added this to allow testing locally without my custom bot.